src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_editor.cljs

```diff
@@ -168,7 +171,17 @@
   {:get-initial-state
    (fn [{:keys [props]}]
      {:config (:config props)
-      :editing? false})
+      :editing? false
+      :sidebar-visible? true})
    :render
    (fn [{:keys [state refs]}]
-     (render-display state refs))})
+     (render-display state refs))
+   :component-did-mount
+   (fn [{:keys [state refs]}]
+     (set! js/window.onscroll
```

Comment:
that's a good point, especially if there are multiple listeners which may be the case as the codebase matures.